deploy-aks/variables.tf

```diff
@@ -1,15 +1,6 @@
-variable "resource_group_name" {
-  description = "Please input resource group name"
-}
 variable "resource_location" {
   description = "Please input resource location"
 }
-variable "useast-minecraft-aksname" {
```

Comment:
dont hardcode location identifier into code if users are allowed to select their preferred region.